en/development/testing.rst

```diff
@@ -31,6 +31,20 @@ To upgrade PHPUnit and its dependencies for your application, execute the follow
 This will install either PHPUnit 4.x or 5.x, depending on your system setup and your
 composer.json configuration.
 
+To be able to work with PHP 7, it is required to install the version 5, and you'll need to setup
```

Comment:
As just few words before there is "PHP (version) 7", I would use "PHPUnit version 5". Slightly less confusing.